### ISSUE TITLE
Spreadsheet export for Projects on Project Listing

### ISSUE THREAD

**Issue by AndreaVogler** [sw360/sw360portal#326]
Component type (OSS, COTS, ...) is missing in the Spreadsheet export

The EXCEL export should also contain the included sub projects as separate rows

**Comment by AndreaVogler** [comment]
This is not really related to 'Excel export for Components' !

First topic is clear.

Some more words to the second topic :
Please filter for projects 'Reuse: DNA' and then create an EXCEL report 'Project with linked Releases'.
There are separate rows for each Release, so far so good, this is excatly what the option 'Project with linked Releases' should do.
However there are two contained projects, they are gathered in column T 'linked projects with relation ship'.
Why not spending one line for each of the linked projects as well, then I have a list of all included objects (releases or projects) on this level ?

**Comment by mcjaeger** [comment]
So, the tickets is about two things:
* adding component type to listing (component type is actually at component not at release that is why it is not in the listing)
* adding a third type of selection in the pull down menue: projects with releases including subprojects.

**Comment by imaykay** [comment]
Hi @mcjaeger - if I understood your last comment correctly, this issue is only about adding the component type to each release (=row) in the spreadsheet export "Projects with linked releases" on the "Projects" page, so for the list of all projects and not for the export of a single project (which has been done in #425 only for the clearing status tab).

**Comment by mcjaeger** [comment]
> To solve last case that component_id comes only once would take some extra effort and I thought it might be no problem - what do you think?
yes, I agree

> And if I understood the conversation correctly, the second topic is not part of this issue, right?
yes, this is allright

**Comment by imaykay** [comment]
Alright, then this feature is testable via PR #542
